### DISCUSSION TITLE
question about low performance and input lag on rg350p

### DISCUSSION THREAD

**Discussion by ogramilik** [eduardofilo/RG350_adam_image#326] [Q&A]
I have an rg350p and I wanted to know how to configure the retroarch to be perfect as I do on a smartphone or on a PC for example, I did not understand how the latency tab works where I configure it to remove imput lag in games and there are also games that remain with bad performance, as if there was lag or something like that, an example was star wars from snes in the first phase of the desert I already noticed how slow it is to move the character and in the second phase the ship is also slow, is there a way to fix this?  I really wanted a tutorial on how to completely configure retroarch on my rg350p before starting any game to be standardized, you know :/

**Comment by eduardofilo** [comment]
Well, for start, a modern cell phone or PC is many times more powerful than an RG350. Then depending on the "weight" of the platform to emulate, the device will have more or less free resources to produce an emulation and response as accurate to the real hardware as possible. SNES is a machine that begins to put the processor of the RG350 in trouble, especially games with chips in the cartridge. As for setup guides for RetroArch, I guess RetroGameCorps's ones are a guarantee of quality.

https://retrogamecorps.com/2022/02/28/retroarch-starter-guide/
https://retrogamecorps.com/2020/12/24/guide-retroarch-on-rg350-and-rg280-devices/
